concepts/literals/introduction.md

```diff
@@ -33,6 +33,15 @@ auto net_worth_carlisle_cullen{46'000'000'000LL}; // int and uint are not enough
 Floating-point numbers usually resolve to `double` during compilation.
 This is a good default case and use-cases with the narrower `float` type are less frequent than the unsigned example above.
 
+~~~~exercism/advanced
+The [C++23 standard][cpp23-literals] introduces more fine grained control with literals like `f128` or `F16`.
+Some compilers allow their own literals.
+GCC permits the usage of `d` as a literal, but this is not forseen in the standard and can hinder cross-platform compatibility.
```

Comment:
```suggestion
GCC permits the usage of `d` as a literal, but this is not foreseen in the standard and can hinder cross-platform compatibility.
```

Using foreseen feels awkward. I think we're just saying `d` isn't in the standard so let's say that. Foreseen makes me think the standard was supposed to include `d` but it wasn't for some reason. That might be misrepresenting the truth.